### PR TITLE
Change RTE dialogs to modals

### PR DESCRIPTION
**Pull request by shucontech**
Signed-off-by: Saksham Gupta <shucontech@gmail.com>

### Description

Please describe your pull request.

Fixes #

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: js/src/database/routines.js [hunk 1]
```diff
@@ -166,6 +166,9 @@ const DatabaseRoutines = {
             if (data.success === true) {
                 // We have successfully fetched the editor form
                 Functions.ajaxRemoveMessage($msg);
+                $('#routineEditorModal').modal('show');
+                $('#routineEditorModalLabel').first().html($this.attr('title') || $this.text() || $(data.title).text());
```
Comment (williamdes) [inline js/src/database/routines.js:170]: I think you could use text?
Not sure how special chars are handled

File: themes/pmahomme/scss/_modal.scss [hunk 2]
```diff
@@ -1,3 +1,8 @@
 .modal-header {
   background-image: linear-gradient(#f8f8f8, #d8d8d8);
 }
+
+#routineEditorModalDialog {
```
Comment (williamdes) [inline themes/pmahomme/scss/_modal.scss:5]: Bootstrap classes could be better than overriding it this way

Comment (shucon) [reply themes/pmahomme/scss/_modal.scss:5]: I wasn't able to fix it using bootstrap, can you please let me know a way for that? I'll try to fix it.

Comment (shucontech) [reply themes/pmahomme/scss/_modal.scss:5]: I wasn't able to fix it using bootstrap, can you please let me know a way for that? I'll try to fix it.

Comment (williamdes) [reply themes/pmahomme/scss/_modal.scss:5]: Hm, not sure what to advice but some `w-` classes could work
`w-100` or something similar if you have give by some way or class a width to the modal

**Comment by shucontech** [comment]
Will complete the PR once the new datetime picker is finalized.

**Pull request closed without merging**